numpy/_core/src/umath/ufunc_object.c

```diff
@@ -6515,6 +6515,12 @@ ufunc_get_signature(PyUFuncObject *ufunc, void *NPY_UNUSED(ignored))
     return PyUnicode_FromString(ufunc->core_signature);
 }
 
+static PyObject *
+ufunc_get_module(PyUFuncObject *ufunc, void *NPY_UNUSED(ignored))
+{
+    return PyUnicode_FromString("numpy");
```

Comment:
Now I think that it isn't right - `numpy.strings` also have ufuncs, could we add `const char *module` to UFunc struct?

Reply:
Solved in #27735